### DISCUSSION TITLE
avd 관련 에러

### DISCUSSION THREAD

**Discussion by lemonit96** [velopert/dealing-with-react-native#12] [Q&A]
안녕하세요. 저번 질문에 답변해주신대로 실리콘 android studio 설치후 교재 p.55 yarn android 명령 실행시 avd 관련 에러로 제대로 실행이 되지 않아서 질문 올립니다. 구글링 해봐도 답을 못찾겠습니다ㅠㅠ

<img width="1440" alt="스크린샷 2021-12-30 오후 10 00 23" src="https://user-images.githubusercontent.com/70015264/147755357-e608ac32-a484-473d-b1e4-8d2f03607ec5.png">

<img width="1440" alt="스크린샷 2021-12-30 오후 10 00 30" src="https://user-images.githubusercontent.com/70015264/147755370-4e3ce0c4-e75a-4666-9101-f9cdf0fc4e8c.png">

마지막 사진은 그냥 에뮬만 실행시켰을때 발생하는 요류입니다. 혹시 에러해결에 도움될까해서 올립니다.

<img width="1440" alt="스크린샷 2021-12-30 오후 10 04 41" src="https://user-images.githubusercontent.com/70015264/147754795-a6730d23-1722-4380-aa53-2a0b251f5145.png">

**Comment by slo-ow** [comment]
해당 프로젝트에서 pod install 다시 해보세요
